frontend/src/components/marketplace/MarketplacePage.jsx

```diff
@@ -24,6 +25,10 @@ function MarketplacePage () {
       <div className="grid-container">
         <div className="grid-child page-container" key = "grid-child page-container">
           <FiltersCollection/>
+          <br></br>
+          <br></br>
+          <br></br>
+          <ControlledCarousel items={items}/>
```

Comment:
To make it distinct (in it's own container with a grey background and border), you can adjust the grid. So the grid would have two small sections on the left.

something like
<div grid-child>
<FiltersCollection />
</div>
<div grid-child>
<ControlledCarousel />
</div>

Reply:
used className='row' and className='col' instead